Containers/nextcloud/Dockerfile

```diff
@@ -1,29 +1,42 @@
 # From https://github.com/nextcloud/docker/blob/master/23/fpm-alpine/Dockerfile
 FROM php:8.0.28-fpm-alpine3.16
 
+VOLUME /var/www/html
+
+ENV PHP_MEMORY_LIMIT 512M
+ENV PHP_UPLOAD_LIMIT 10G
+ENV PHP_MAX_TIME 3600
+
+ENV NEXTCLOUD_VERSION 25.0.6
+
+COPY *.sh upgrade.exclude /
+COPY config/* /usr/src/nextcloud/config/
+
+ENV NEXTCLOUD_UPDATE=1
+
+COPY supervisord.conf /supervisord.conf
+
+COPY --chmod=775 start.sh /start.sh
+COPY --chmod=775 notify.sh /notify.sh
+COPY --chmod=775 notify-all.sh /notify-all.sh
```

Comment:
why are some files copied directly and some by using *,sh?

Reply:
because of inconsistence... Better would be copying them with *.sh of course if possible...

Reply:
```suggestion
COPY --chmod=775 *.sh /
COPY upgrade.exclude /upgrade.exclude
COPY config/* /usr/src/nextcloud/config/

ENV NEXTCLOUD_UPDATE=1

COPY supervisord.conf /supervisord.conf
```

Reply:
done